### DISCUSSION TITLE
Using command line Parameters

### DISCUSSION THREAD

**Discussion by billsecond** [oobabooga/text-generation-webui#4841] [Q&A] [answered]
I am trying to launch the server using the following commands:

start_windows.bat --listen --listen-port 80 --model-dir D:\gpt\text-generation-webui-main\models\Mistral-7B-Instruct-v0.1 --multi-user

The --listen, --listen-port and --multi-user work, but it is not loading the model.  Can you tell me why the model-dir is not working?
Also, how do I specify a preset?  

I want to host this chat internally and I want my team to be able to use it.

**Comment by TheLounger** [comment, marked as answer]
https://github.com/oobabooga/text-generation-webui#basic-settings

`--model-dir MODEL_DIR` | Path to directory with all the models.

You probably want `--model`, e.g.
```start_windows.bat --listen --listen-port 80 --model Mistral-7B-Instruct-v0.1 --multi-user```

As for the preset, you can set it in `settings.yaml` or `models/config-user.yaml`.

**Reply by billsecond** [reply]
YES, that worked exactly!
